ci/gradle.properties

```diff
@@ -0,0 +1,2 @@
+org.gradle.jvmargs=-Xmx16g -XX:+UseGCOverheadLimit -XX:GCTimeLimit=10 -Dfile.encoding=UTF-8
```

Comment:
Do we really need 16G?

Reply:
I'll do some testing.
An idea for this change was to specify CI xmx separately from local, and that we have more than enough ram per build agent